Documenting `nargo preprocess` (#79)
noir-lang/book: src/getting_started/nargo/commands.md

Generate a Solidity verifier smart contract for the program.

> ## `nargo preprocess <acir_file>`
> 
> Accepts an ACIR file and generate proving and verification keys from it.

Reviewer: ```suggestion
## `nargo preprocess <build_artifact>`

Generate proving and verification keys from a build artifact file.
```

Switching this to the imperative form to match other commands.
